.github/workflows/test.yml

```
@@ -0,0 +1,30 @@
+name: Run Tests
+
+on:
+  push:
+    branches: [ main, 97-pytest ]
```

Accept right before merge, when no longer need for testing

```suggestion
    branches: [ main ]
```